SUPPORT.md

```diff
@@ -0,0 +1,67 @@
+# Payara Support
+
+The [Payara Foundation](http://www.payara.org) is a non-profit foundation behind the development of Payara Server and Payara Micro (the Payara Platform). The Payara Foundation continues to develop the Payara Platform based on the needs of our support customers.
```

Comment:
Payara Foundation doesn't have any customer. So combined with the comment below commercial support should be introduced by Payara Services Limited